### PR TITLE
Linted markdown for readability

### PR DESCRIPTION
**Pull request by rogersachan**
I've linted the markdown of the `COMMANDS.md` file so it is easier to edit, maintain, and render.

### CODE REVIEW

**Comment by Zarel** [comment]
You've changed the heading standard and the code-block standard, but it's unclear to me why your standards are better.

**Comment by Zarel** [comment]
In fact, I think the current format is vastly better for plaintext readability, which is the main reason I like Markdown in the first place.

**Comment by rogersachan** [comment]
@Zarel I've done the linting to the tune of https://github.com/mivok/markdownlint

**Comment by rogersachan** [comment]
The linter exists for ease of use and so that reading and writing markdown is easier.  For example, you use 72 dashes/equal signs for headers (I understand you can copy/paste, but doing this with every markdown document that may be used in this project doesn't really make sense)

TL;DR Easier to read, easier to write and reasonable at the same time.

**Comment by rogersachan** [comment]
Indented code blocks cannot be highlighted, so I had to go and add the ` ``` ` so the syntax could be colorized.

**Comment by Zarel** [comment]
I'd agree that it would make writing Markdown easier, but I disagree that it would make reading easier.

And since the documentation is read far more often than it's written, I think optimizing for readability is more important.

For example, 72 dashes for headers may be annoying to write, but it makes it a lot easier to identify section breaks when you're scrolling around the document.
